crates/bevy_ecs/src/entity/map_entities.rs

```diff
@@ -8,6 +8,32 @@ pub enum MapEntitiesError {
     EntityNotFound(Entity),
 }
 
+/// Operation to map all contained [`Entity`](crate::entity::Entity) fields in
```

Comment:
```suggestion
/// Operation to map all contained [`Entity`] fields in
```
Because `Entity` is in scope for this module, we don't need to clarify the path.